cirq-core/cirq/experiments/google_v2_supremacy_circuit.py

```diff
@@ -16,9 +16,14 @@
 from typing import Callable, Iterable, TypeVar, cast, Sequence
 
 from cirq.circuits import InsertStrategy
-from cirq import circuits, devices, ops
+from cirq import _compat, circuits, devices, ops
 
 
+@_compat.deprecated(
+    deadline='v0.16',
+    fix='Install ReCirq and use '
+    + 'recirq.beyond_classical.generate_boixo_2018_beyond_classical_v2',
```

Comment:
think there is a pre to use parens for multiline strings instead of +

Reply:
Done.